### PR TITLE
add bear65 V2 keyboard

### PR DESCRIPTION
**Pull request by 704340378**
## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bear65/config.h [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#pragma once
+
+#include "config_common.h"
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x45D4
+#define PRODUCT_ID      0x0428
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    MT
+#define PRODUCT         Bear65 V2
```
Comment (drashna) [inline keyboards/bear65/config.h:26]:
```suggestion
```

USB device info needs to be moved to the info.json.

File: keyboards/bear65/config.h [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
```
Comment (drashna) [inline keyboards/bear65/config.h:58]: This is the default behavior, and can be removed. 
```suggestion
```

File: keyboards/bear65/info.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "keyboard_name": "bear65 V2", 
+    "url": "https://github.com/704340378/qmk_firmware/tree/master/keyboards/bear65", 
+    "maintainer": "qmk",
```
Comment (drashna) [inline keyboards/bear65/info.json:4]:
```suggestion
    "maintainer": "MT", 
    "manufacturer": "MT", 
    "usb": {
        "device_version": "0.0.1",
        "pid": "0x042B",
        "vid": "0x45D4"
    },
```

File: keyboards/bear65/rules.mk [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes	# Audio control and System control(+450)
+CONSOLE_ENABLE = yes	# Console for debug(+400)
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/bear65/rules.mk:14]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/bear65/rules.mk [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes	# Audio control and System control(+450)
+CONSOLE_ENABLE = yes	# Console for debug(+400)
```
Comment (drashna) [inline keyboards/bear65/rules.mk:11]:
```suggestion
BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes	# Mouse keys
EXTRAKEY_ENABLE = yes	# Audio control and System control
CONSOLE_ENABLE = yes	# Console for debug
```

File: keyboards/bear65/rules.mk [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
```
Comment (drashna) [inline keyboards/bear65/rules.mk:13]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bear65/config.h (hunk 1 above)
Comment (704340378) [reply keyboards/bear65/config.h]: i dont know how to add it in info.json

**Pull request closed without merging**